lib/cdo/graphics/certificate_image.rb

```diff
@@ -105,10 +105,14 @@ def create_workshop_certificate_image(image_path, fields)
     string = field[:string].to_s
     next if string.empty?
 
-    y = field[:y] || 0
+    # The x,y position to center the text at.
     x = field[:x] || 0
+    y = field[:y] || 0
+    # The width/height in pixels of the bounding box for the text.
+    width = field[:width]
+    height = field[:height]
     pointsize = field[:pointsize] || 70
-    apply_text(background, string, pointsize, 'Times bold', 'rgb(87,87,87)', x, y, CERT_NAME_AREA_WIDTH, CERT_NAME_AREA_HEIGHT)
+    apply_text(background, string, pointsize, 'Times bold', 'rgb(87,87,87)', x, y, width, height)
```

Comment:
This was the specific line which was causing the teacher names to be "smaller" on the workshop certificates. Technically in pixels, the names were the same size as other certificates; however, the workshop certificate is a significantly higher resolution, so the image is much more pixel dense and the names are smaller relative to the whole certificate.